internal/domain/filter.go

```diff
@@ -309,12 +309,26 @@ func (f Filter) CheckFilter(r *Release) ([]string, bool) {
 		r.addRejectionF("year not matching. got: %d want: %v", r.Year, f.Years)
 	}
 
-	if f.MatchCategories != "" && !contains(r.Category, f.MatchCategories) {
-		r.addRejectionF("category not matching. got: %v want: %v", r.Category, f.MatchCategories)
+	if f.MatchCategories != "" {
+		var categories []string
+		categories = append(categories, r.Categories...)
+		if r.Category != "" {
+			categories = append(categories, r.Category)
```

Comment:
did you intend to do anything with this? 😺

Reply:
It's there so we can use the `containsAny` to check a slice of strings. Maybe it can be rewritten with an inline append.